payload/reggae/dcompile.d

```diff
@@ -110,6 +128,33 @@ private string[] mapToLdcOptions(in string[] compArgs) @safe pure {
 }
 
 
+private auto invokeCompiler(in string[] args, in string objFile) @safe {
+    version(Windows) {
+        static string quoteArgIfNeeded(string a) {
+            return !a.canFind(' ') ? a : `"` ~ a.replace(`"`, `\"`) ~ `"`;
+        }
+
+        const rspFileContent = args[1..$].map!quoteArgIfNeeded.join("\n");
+
+        // max command-line length (incl. args[0]) is ~32,767 on Windows
+        /*if (rspFileContent.length > 32_000)*/ {
```

Comment:
?

Reply:
Just temporarily commented out to have dcompile's output response file tested via CI with both DMD and LDC - there are apparently no long cmdlines in the testsuite.

Reply:
Ah, ok. Please revert.